index.js

```diff
@@ -327,3 +325,22 @@ const loadProjects = () => {
 };
 
 loadProjects();
+
+// email validation start
+
+const form = document.querySelector('form');
+const emailInput = document.querySelector('#email');
+const errorText = document.querySelector('#error-text');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailText = emailInput.value;
+
+  if (emailText === emailText.toLowerCase()) {
+    form.submit();
+  } else {
+    errorText.style.display = 'block';
```

Comment:
- [ ] Good job so far 👍🏽 . I think it will be better if the error message shows up close to the submit button or the input fields. (i.e error message can show up around the email input field or just around the submit button)
![image](https://github.com/devendra-alt/portfolio/assets/63473584/ff41417d-77da-466e-b05a-ec85b360bf53)

Reply:
hello @vickymarz, thank you for your suggestion,  [here](https://drive.google.com/file/d/1LnHDc9y2VnooIYNIb0HVmqfWdSDD_LvO/view?usp=sharing) is the attachment of my latest update on the form error message position.